deployment/terraform/create.go

```diff
@@ -32,6 +32,9 @@ type Terraform struct {
 // terraformOutput contains the output variables which are
 // created after a deployment.
 type terraformOutput struct {
+	ProxyIP struct {
+		Value []string
+	} `json:"proxyIP"`
```

Comment:
I think I will need this to setup agents :)

Reply:
It should not really block your progress. Right now, you can go ahead with one server and just use `Instances.Value[0].PublicIP` as your target.